static/js/apps/biomed_nl/app.tsx

```diff
@@ -86,23 +102,125 @@ export function App(): ReactElement {
   const [answer, setAnswer] = useState<DisplayedAnswer>(null);
   const [showLoading, setShowLoading] = useState<boolean>(false);
 
+  // A loading spinner that shows new biomed pun every minute
+  function SpinnerWithText(): ReactElement {
```

Comment:
Let's move this ReactElement into its own file

Reply:
moved!